consensus/core/src/block.rs

```diff
@@ -86,8 +86,8 @@ impl BlockAPI for BlockV1 {
         self.author
     }
 
-    fn timestamp(&self) -> BlockTimestampMs {
-        self.timestamp
+    fn timestamp_ms(&self) -> BlockTimestampMs {
```

Comment:
Although I am always in favour of the prescriptive naming convention, the returned type `BlockTimestampMs` is already bearing the information that the returned timestamp is in milliseconds.

Reply:
I think type info may be unavailable at some callsites.